nebullvm/installers/installers.py

```diff
@@ -164,39 +227,92 @@ def install_openvino(with_optimization: bool = True):
     cmd = ["pip3", "install", "scipy>=1.7.3"]
     subprocess.run(cmd)
 
+    try:
+        from openvino.runtime import (  # noqa F401
+            Core,
+            Model,
+            CompiledModel,
+            InferRequest,
+        )
+    except ImportError:
+        return False
+
+    return True
+
 
 def install_onnxruntime():
     """Helper function for installing the right version of onnxruntime."""
     distribution_name = "onnxruntime"
-    if torch.cuda.is_available():
+    if gpu_is_available():
         distribution_name = f"{distribution_name}-gpu"
     if _get_os() == "Darwin" and get_cpu_arch() == "arm":
         cmd = ["conda", "install", "-y", distribution_name]
     else:
         cmd = ["pip3", "install", distribution_name]
     subprocess.run(cmd)
     # install requirements for onnxruntime.transformers
-    cmd = ["pip3", "install", "coloredlogs", "sympy"]
+    cmd = ["pip3", "install", "coloredlogs", "sympy", "transformers"]
```

Comment:
I'm not sure we should install transformers every time ORT is installed. I would prefer to have the double option `onnxruntime` and `ort-transformers`.